lib/wit.rb

```diff
@@ -59,6 +59,24 @@ def interactive(handle_message=nil, context={})
     puts
   end
 
+  def create_new_app(payload, set_new_app_token = false)
+    response = req(logger, @access_token, Net::HTTP::Post, "/apps", {}, payload)
+    @access_token = response['access_token'] if set_new_app_token
+    return response
+  end
+
+  def train(payload)
```

Comment:
can you rename this to `post_utterances` instead of `train`? it would make it clear that this function is for adding utterances, as there are other ways for an app to trigger training as well

Reply:
ok. i will change it, also i dont know another way except to train the app using [wit.ai](url) website.